### PR TITLE
Add test for signing an add-on with a unicode guid.

### PR DESCRIPTION
**Pull request by EnTeQuAk**
Fixes mozilla/addons#301 (or rather, makes sure that it works)

### CODE REVIEW

File: src/olympia/lib/crypto/tests/test_packaged.py [hunk 1]
```diff
@@ -257,7 +245,6 @@ def test_call_signing(self):
             'SHA256-Digest: 3Wjjho1pKD/9VaK+FszzvZFN/2crBmaWbdisLovwo6g=\n\n'
         )
 
-    @responses.activate
```
Comment (eviljeff) [inline src/olympia/lib/crypto/tests/test_packaged.py]: Out of interest, what did `@responses.activate` do why don't we need it any more?

Comment (EnTeQuAk) [reply src/olympia/lib/crypto/tests/test_packaged.py]: I recently activated support for blocking calls to the web in our test suite so that tests that actually need to call an outside server have to activate it. That's happening via responses.activate :)
